Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -373,42 +361,20 @@ public extension UIView {
     /// SwifterSwift: Rotate view by angle on relative axis.
     ///
     /// - Parameters:
-    ///   - angle: angle to rotate view by.
-    ///   - type: type of the rotation angle.
-    ///   - animated: set true to animate rotation (default is true).
-    ///   - duration: animation duration in seconds (default is 1 second).
-    ///   - completion: optional completion handler to run with animation finishes (default is nil).
+    ///   - angle: Angle to rotate view by.
+    ///   - animated: Set `true` to animate rotation (default is `true`).
```

Comment:
Is this the default? 😂

Reply:
Ok, you got me coping and pasting the comment from the deprecated function 😂

It looks like this typo has been a long time. I'll fix them. 😁